lib/worker/line-numbers.js

```diff
@@ -0,0 +1,90 @@
+function parse(file) {
+	const fs = require('fs');
+	const acorn = require('acorn');
+	const walk = require('acorn-walk');
+
+	const ast = acorn.parse(fs.readFileSync(file, 'utf8'), {
```

Comment:
Did you try to use the `acorn` parser of `recast`? I'm thinking maybe it's the `esprima` parser which handles tabs funkily. Might be wrong, though, haven't checked/tested.

The reason I'm asking is I don't see how we would handle `typescript`/`babel` now?

Plus, you wouldn't need `acorn-walk`, nor do the sorting, right?

Reply:
`recast` is designed for rewriting code, but all we need is the locations of call expressions. `@ava/babel` already has access to the Babel parser, and we can do something with `@ava/typescript` as well.

Reply:
Fair enough!